### PR TITLE
flex panel gallery

### PR DESCRIPTION
**Pull request by iemeka**

### CODE REVIEW

File: flex-panel/index-START.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      <p>Motion</p>
+    </div>
+  </div>
+
+  <script>
+    
+    const panels = document.querySelector(".panels");
+
+    function toggleOpen(e){
+      var targetPanel = e.path[0].tagName === "DIV" ? e.path[0] : e.path[1]
```
Comment (dokasto) [inline flex-panel/index-START.html]: Since `targetPanel` is not going to change make it a `const`

File: flex-panel/index-START.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      targetPanel.classList.toggle('open');
+    }
+
+    function toggleActive(e){
+      if (e.propertyName.includes('flex')){
+        let targetPanel = e.path[0].tagName === "DIV" ? e.path[0] : e.path[1]
```
Comment (dokasto) [inline flex-panel/index-START.html]: Since `targetPanel` is not going to change make it a `const`

**Comment by dokasto** [comment]
@iemeka also rename the file from `index-START.html` => `index.html`

File: flex-panel/index-START.html (hunk 1 above)
Comment (iemeka) [reply flex-panel/index-START.html]: okay

**Comment by iemeka** [comment]
renamed.

**Review by dokasto — APPROVED** [review]
alt text

**Comment by iemeka** [comment]
lol

**Pull request merged**